A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/AltimeterIndicator.js

```diff
@@ -441,6 +441,8 @@ class Jet_PFD_AltimeterIndicator extends HTMLElement {
         }
 
         const groundReference = indicatedAltitude - Simplane.getAltitudeAboveGround();
+        SimVar.SetSimVarValue("L:A32NX_ALTIMETER_GROUND_REFERENCE", "feet", groundReference);
```

Comment:
Why are we setting this simvar if we could just use the same calculation instead of the simvar ?

Reply:
How much does it impact on performance?
I thought that setting it from where it was calculated and then picking it up where it was needed would not give problems of any kind

Reply:
Setting a simvar inside an instrument view is bad for separation of concerns.

If you wanna create this simvar, you should create an `A32NX_GroundReference.js` module in `A32NX_Core`.

Reply:
Interesting ... I moved the ground reference logic into its specific module it can be useful to centralize this part. Can you check if it's correct?
Another question, is it right that the core part scripts are included inside "A320_Neo / CDU / A320_Neo_CDU.html" ?? Sorry for my bad English, it's my big problem :(